pkg/noderesourcetopology/least_numa.go

```diff
@@ -0,0 +1,173 @@
+package noderesourcetopology
```

Comment:
let's add the copyright boilerplate. Feel free to copy it from any other source file, but please make sure copyright year is 2023

Reply:
👍